packages/node/src/plugins/LoadFileChunkLoadingRuntimeModule.ts

```diff
@@ -348,7 +348,7 @@ class ReadFileChunkLoadingRuntimeModule extends RuntimeModule {
                      */
                             `var requestedRemote = remoteRegistry[${JSON.stringify(
                               name
-                            )}]`,
+                            )}];`,
```

Comment:
according to the variable evaluation standards of this file, you need to add a semicolon to make sure that webpack evaluates the javascript file properly.